src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/NonGCStaticsNode.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 using System.Collections.Generic;
-
+using ILCompiler.Compiler.Compiler.DependencyAnalysis;
```

Comment:
These will go away once you change the namespace - please make sure to leave an empty line - we separate system namespaces although it's not enforced by the CI (there is a rule for it but we don't run formatter like linker repo).